### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by JakobTernes** [miho/MonacoFX#31]
First of all, I really like this project!
It looks like it is currently not maintained, though. What is the current status of this project? Are contributions welcome?

I would like to add a setFontSize method to the Editor. I think https://github.com/miho/MonacoFX/pull/27 is a great addition too.

Maybe someone from the community is willing to maintain the project, if you are fully occupied with other projects (if you would be ok with that)?

Cheers

**Comment by EasyG0ing1** [comment]
@JakobTernes I agree with you. When it comes to being able to throw a code editor inside of a Java project, the options out there are pretty thin and they can be rather overwhelming using some of the other libraries. This library is simple to use and robust enough to meet basic needs use cases. Though there is room for improvement in this library as well and having it actively developed / maintained would be amazing.

I'm just adding my opinion and think your suggestion is a good one.
